pkg/kubelet/types/constants.go

```diff
@@ -43,8 +43,8 @@ const (
 // entries here should be moved to staging/src/k8s.io.api/core/v1/types.go
 // once the feature managing the condition graduates to Beta.
 const (
-	// PodHasNetwork indicates networking has been configured successfully for the
+	// PodReadyToStartContainers indicates networking has been configured successfully for the
 	// pod and IP address(es) assigned. Images for containers specified in the pod
 	// spec can be pulled and containers launched after this condition is true.
-	PodHasNetwork = "PodHasNetwork"
+	PodReadyToStartContainers = "PodReadyToStartContainers"
```

Comment:
See comment above about shifting this to staging/src/k8s.io.api/core/v1/types.go if we decide to go to Beta right now.

Reply:
We will keep in Alpha for now.